politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -200,6 +227,159 @@ func addCreditsAction() error {
 	return nil
 }
 
+func replayCommentsJournal(path string, pubkeys map[string]struct{}) error {
```

Comment:
Is this shared code?

Reply:
No, this is a lightweight version of the gitbe code.